### PR TITLE
DOC-122/Document EnterpriseGraphs

### PR DESCRIPTION
**Pull request by nerpaula**
preview: https://deploy-preview-1054--zealous-morse-14392b.netlify.app/docs/3.10/graphs-enterprise-graphs.html

Remaining points:

- [x] add example for `arangoimport` / `arangoexport`
- [x] add screenshot for Create Graph dialog (UI still in progress)
- [x] adjust all arangosh code examples + docublocks regeneration

### CODE REVIEW

**Comment by ansoboleva** [comment]
This PR requires examples regeneration for 3.10 because of the new docublocks.

**Review by mchacki — APPROVED** [review]
LGTM thank you for writing this.!!

Just one minor correction

File: 3.10/graphs-enterprise-graphs-getting-started.md [hunk 1]
```diff
@@ -29,6 +29,87 @@ the *_key* values from edges. The `enterprise-graph` module does not allow
...
+so we can again just import the data, and just target the new collection name:
+
+    arangoimport --collection vertices --file docOutput/old_vertices.jsonl
+
+For the edges we need to apply more changes, as we need to rewire them.
+First thing, we again have to remove the `_key` value as it is disallowed for enterprise graphs.
+Second, we have changed the name of the `_from` collection, so we need to provide a `from-collection-prefix`.
+Third, the same for the `_to` collection. Note here: We can only change all vertices on `_from` resp. `_to`
+side with this mechanism. However we can use different collections on `_from` and `_to`.
+Fourth, in order to make the change of vertex collection actually happy we need to allow "overwrite-collection-prefix"
```
Comment (Simran-B) [inline 3.10/graphs-enterprise-graphs-getting-started.md]:
```suggestion
Fourth, in order to make the change of vertex collection actually happy we need to allow `--overwrite-collection-prefix`
```

File: 3.10/graphs-enterprise-graphs-getting-started.md [hunk 2]
```diff
@@ -29,6 +29,87 @@ the *_key* values from edges. The `enterprise-graph` module does not allow
...
+if this flag is not set, only values without any given collection will be changed, this is helpful if your data is not
```
Comment (Simran-B) [inline 3.10/graphs-enterprise-graphs-getting-started.md]:
```suggestion
if this option is not set, only values without any given collection are changed. This is helpful if your data is not
```

File: 3.10/graphs-enterprise-graphs-getting-started.md [hunk 3]
```diff
@@ -29,6 +29,87 @@ the *_key* values from edges. The `enterprise-graph` module does not allow
...
+as it cannot be parsed line by line. The `csv` and `tsv` formats are also fine,
+but require to define the list of attributes to export. `jsonl` exports data
+as is, and due to it's line based layout can be processed line by line and
+therefore has no artifical restrictions on the data.
+After this step we there are two files in `docOutput` folder, that should look
+like this:
+
+1) `docOutput/old_vertices.jsonl`:
+
+    {"_key":"Alice","_id":"old_vertices/Alice","_rev":"_edwXFGm---","attribute1":"value1"}
```
[the remark this answers is not in the document]
Comment (Simran-B) [reply 3.10/graphs-enterprise-graphs-getting-started.md]: Text in a list that wraps around should be indented like the first character after the bullet point, and fenced code blocks should be indented like this too, including the triple backticks. Adjusted by https://github.com/arangodb/docs/pull/1054/commits/2a57778de76f0471dcb267f0b169dc0b6b95900d

**Pull request merged**